### ISSUE TITLE
megaCMD and systemd units?

### ISSUE THREAD

**Issue by marcoszy** [meganz/MEGAcmd#750]
Hi there,

I noticed that megaCMD (and megaCMD server) don't starts automatically when I boot my machine. Today, for example, I needed to run mega-cmd command in order it be started resume my backup.

Is there a systemd unit I can use to do that?

Thanks in advance.

**Comment by polmr** [comment]
No, there is not installed via MEGA CMD. However [here](https://github.com/meganz/MEGAcmd/issues/88#issuecomment-751341956) there's a possible option for you

**Comment by marcoszy** [comment]
Many thanks.

Do I have to handle only with mega-cmd-server at startup or is it necessary to also start mega-cmd itself?
